spy_terminal.py

```diff
@@ -27,76 +30,94 @@ def main():
     while x != 'boom':
         if welcome():
             password = set_up()
-            scrambled_message = new_message(codex,count)
+            scrambled_message = new_message(codex, count)
             count = 1
         else:
-            retrieve_message(password, scrambled_message,codex,count)
+            retrieve_message(password, scrambled_message, codex, count)
             scrambled_message = None
             count = 0
         x = input("To self-destruct type 'boom'. Otherwise, hit enter. ")
-    self_destruct(5,'KABOOM', "Yes")
+    self_destruct(5, 'KABOOM', "Yes")
+
 
 '''---------------------SUPPORTING FUNCTIONS ------------------------'''
```

Comment:
Having a comment like `supporting functions` is a good suggestion that you probably have too much code in a single file. Instead of dumping everything in one file, pull out the reusable pieces in a `lib` or `utils` file or directory (if there is further categorization that would be helpful), and then import them here